xwiki-platform-core/xwiki-platform-livetable/xwiki-platform-livetable-ui/src/main/resources/XWiki/LiveTableResultsMacros.xml

```diff
@@ -764,28 +775,35 @@
       #set ($matchType = 'partial')
     #end
   #end
-  #set ($filterQuery = "#livetable_getFilterQuery(""${safe_tableAlias}.value"" $matchType)")
+  #set ($filterQuery = "#livetable_getFilterQuery(""${safe_tableAlias}.value"" $matchType false $multilistsize)")
   #set ($whereSql = "${whereSql} and $filterQuery.trim()")
-  #livetable_addFilterParam($filterValue $matchType $whereParams)
+  #livetable_addFilterParam($filterValue $matchType $whereParams $multilistsize)
 #end
 
 
-#macro (livetable_getFilterQuery $column $matchType $isList)
+#macro (livetable_getFilterQuery $column $matchType $isList $multilistsize)
   #if ($matchType == 'partial' || $matchType == 'prefix')
     upper($column) like upper(?)
   #elseif ($isList)
     ? in elements($column)
+  #elseif (($multilistsize > 1) && ($matchType == 'exact'))
```

Comment:
> Am I wrong?

Yes. Wiki pages should be modified on a running XWiki instance, then exported, then copied to the source tree, and then formatted with ``mvn xar:format``. If you modify wiki pages directly in the source tree then:
* you always forget about XML encoding
* you don't see the effect of your changes on a running XWiki instance.